colibre/auto_plotter/snia.yml

```diff
@@ -0,0 +1,247 @@
+stellar_mass_snia_rates_50:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "snia_rates.snia_rates_50_kpc"
+    units: "1/year"
+    start: 1e-8
+    end: 1e-1
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate relation (50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+stellar_mass_snia_rates_per_stellar_mass_50:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "apertures.mass_star_50_kpc"
+    units: Solar_Mass
+    start: 1e6
+    end: 1e12
+  y:
+    quantity: "derived_quantities.snia_rate_per_stellar_mass_50_kpc"
+    units: "1/year/Solar_Mass"
+    start: 1e-15
+    end: 1e-11
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e6
+      units: Solar_Mass
+    end:
+      value: 1e12
+      units: Solar_Mass
+  metadata:
+    title: Stellar Mass-SNIa rate per stellar mass relation (50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+star_formation_rates_snia_rates_per_stellar_mass_50:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "apertures.sfr_gas_50_kpc"
+    units: "Solar_Mass/year"
+    start: 1e-3
+    end: 1e2
+  y:
+    quantity: "derived_quantities.snia_rate_per_stellar_mass_50_kpc"
+    units: "1/year/Solar_Mass"
+    start: 1e-15
+    end: 1e-11
+  median:
+    plot: true
+    log: true
+    adaptive: true
+    number_of_bins: 30
+    start:
+      value: 1e-3
+      units: "Solar_Mass/year"
+    end:
+      value: 1e2
+      units: "Solar_Mass/year"
+  metadata:
+    title: SFR-SNIa rate per stellar mass relation (50 kpc aperture)
+    caption: Uses a 50 kpc 3D aperture.
+    section: SNIa Rate
+
+specific_star_formation_rates_snia_rates_per_stellar_mass_50:
+  type: "scatter"
+  select_structure_type: 10
+  comment: "Centrals only"
+  legend_loc: "upper left"
+  x:
+    quantity: "derived_quantities.specific_sfr_gas_50_kpc"
+    units: "1/(1e9*year)"
```

Comment:
```suggestion
    units: "1/gigayear"
```
Otherwise the units look strange

![Screenshot from 2022-09-27 14-21-10](https://user-images.githubusercontent.com/20153933/192525790-3349baa6-1950-4ee1-94c6-9982b8bfdd6a.png)